tasks.py

```diff
@@ -226,3 +265,17 @@ def publish(c, tag):  # pylint: disable=unused-argument
     LOG.info("Pushing %s to docker hub...", repository)
     CLIENT.images.push(repository=repository)
     LOG.info("Done publishing the %s Docker image", repository)
+
+
+@task
+def update(c):  # pylint: disable=unused-argument
+    """Update the goat dependencies"""
+    repo = "github/super-linter"
+    version = get_latest_release_from_github(repo=repo)
+    update_dockerfile_from(image=repo, tag=version)
+
+    try:
+        subprocess.run(["pipenv", "update"], capture_output=True, check=True)
```

Comment:
Recommend printing the output so it's clear that the function ran, otherwise there's no indication of what occurred.

```suggestion
        subprocess.run(["pipenv", "update"], capture_output=False, check=True)
```